code/game/objects/items/devices/traitordevices.dm

```diff
@@ -264,6 +264,28 @@ effective or pretty fucking useless.
 		GLOB.active_jammers -= src
 	update_icon()
 
+/obj/item/storage/toolbox/emergency/turret
+	desc = "You feel a strange urge to hit this with a wrench."
+
+/obj/item/storage/toolbox/emergency/turret/PopulateContents()
+	new /obj/item/screwdriver(src)
+	new /obj/item/wrench(src)
+	new /obj/item/weldingtool(src)
+	new /obj/item/crowbar(src)
+	new /obj/item/analyzer(src)
+	new /obj/item/wirecutters(src)
+
+/obj/item/storage/toolbox/emergency/turret/attackby(obj/item/I, mob/living/user, params)
+    if(I.tool_behaviour == TOOL_WRENCH && user.a_intent == INTENT_HARM)
+        user.visible_message("<span class='danger'>[user] bashes [src] with [I]!</span>", \
+            "<span class='danger'>You bash [src] with [I]!</span>", null, COMBAT_MESSAGE_RANGE)
+        playsound(src, "sound/items/drill_use.ogg", 80, TRUE, -1)
+        var/obj/machinery/porta_turret/syndicate/pod/toolbox/turret = new(get_turf(loc))
+        turret.faction = list("[REF(user)]")
+        qdel(src)
+
+    ..()
```

Comment:
You may wanna early return after doing the qdel in the above if, considering otherwise it'll call the parent proc on something qdel()d, which ain't too good.

Reply:
return added